src/main/java/seedu/address/model/student/ClassCode.java

```diff
@@ -11,7 +11,7 @@
 
     public static final String MESSAGE_CONSTRAINTS = "ClassCode can take any values, and it should not be blank";
```

Comment:
perhaps can specify the constraints here, like a single word with alphanumeric characters or something like that